src/Model/ModelManager.php

```diff
@@ -761,10 +761,19 @@ public function modelTransform($class, $instance)
         return $instance;
     }
 
+    /**
+     * NEXT_MAJOR: Remove this method.
+     *
+     * @deprecated since sonata-project/doctrine-orm-admin-bundle 3.x and will be removed in version 4.0.
+     */
     public function modelReverseTransform($class, array $array = [])
     {
+        @trigger_error(sprintf(
+            'Method %s() is deprecated since sonata-project/doctrine-orm-admin-bundle 3.x and will be removed in version 4.0.',
```

Comment:
```suggestion
            'Method "%s()" is deprecated since sonata-project/doctrine-orm-admin-bundle 3.x and will be removed in version 4.0.'
            .' Use "reverseTransform()" instead.',
```